app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -83,9 +83,12 @@ def factory_class
            else
              Bulkrax.default_work_type
            end
+
+      # return the name of the collection
       fc.downcase.camelcase.constantize
```

Comment:
this line was updated from "fc.constantize" to "fc.downcase.camelcase.constantize" for the collection work. the camelcase method doesn't actually camelcase the work though, it titlecases it. (refer to screenshot in pr summary)

Reply:
`fc.downcase.camelcase.constantize || fc.constantize` still throws the no method error

Reply:
@alishaevn do you still get the no method error if you do this? 

```ruby
fc&.downcase&.camelcase&.constantize || fc.constantize
```

Reply:
didn't try that one. I can in a bit though

Reply:
Actually, I don't think my suggestion will work. 

I made the change to `fc.downcase.camelcase.constantize` in order to handle when the model isn't uppercased and/or is snakecase: 

```
work
generic_work
```

But it seems to be tripping up on classes that are already camelcase or titlecased: 

```
BookContributor
bookContributor
```

I have an idea, give me a few minutes to test it

Reply:
```ruby
fc.gsub!(' ', '_')
fc.downcase! if fc =~ /[-_]/
fc.camelcase.constantize
```

I tested this for the following values:

```
genericWork
GenericWork
generic work
Generic work
Generic Work
generic_work
Generic_Work
```

I think something like this would be preferable to the current diff because if `fc.constantize` fails, I think we would get stuck in an infinite loop (infinitely catching and throwing `NameError`s)